Source/Scene/Scene.js

```diff
@@ -1397,6 +1411,10 @@ define([
             commands = frustumCommands.commands[Pass.TRANSLUCENT];
             commands.length = frustumCommands.indices[Pass.TRANSLUCENT];
             executeTranslucentCommands(scene, executeCommand, passState, commands);
+
+            var pickDepth = getPickDepth(scene, context, index);
+            pickDepth.update(context, globeDepth.framebuffer.depthStencilTexture);
+            pickDepth.executeCopyDepth(context, passState);
```

Comment:
We didn't implement the MRT optimization right?  Thinking about it more...MRT _could_ be slower, for example, if the scene has a high depth complexity it would result in a lot more memory bandwidth.  I don't think this warrants a roadmap issue, perhaps add `PERFORMANCE_IDEA` comment in case we run into this extra pass when profiling?